jetson/nav/search/searchStateMachine.cpp

```diff
@@ -224,8 +311,15 @@ NavState SearchStateMachine::executeTurnToTarget()
         }
         return NavState::TurnToTarget;
     }
+    double gimbalAddition = 0.0;
+    if ( mRoverConfig[ "search" ][ "useGimbal" ].GetBool() ){
+        gimbalAddition = mRover->gimbal().getYaw();
+        mRover->gimbal().setDesiredGimbalYaw(mRover->roverStatus().leftCacheTarget().bearing);
+        mRover->publishGimbal();
```

Comment:
why does it use the cached target here but not in searchGateStateMachine??? this is what i mean by inconsistencies!